backend/src/main/java/com/google/models/Alert.java

```diff
@@ -79,8 +80,9 @@ public Entity toEntity() {
     alertEntity.setProperty(Timestamp.TIMESTAMP_PROPERTY, timestampDate.toString());
     alertEntity.setProperty(STATUS_PROPERTY, status.name());
 
-    List<EmbeddedEntity> list = new ArrayList<EmbeddedEntity>();
-    anomalies.forEach(anomaly -> list.add(anomaly.toEmbeddedEntity()));
+    List<EmbeddedEntity> list = anomalies.stream()
+        .map(anomaly -> anomaly.toEmbeddedEntity())
```

Comment:
nit: map(Anomaly::toEmbeddedEntity)

Reply:
thanks for the suggestion! this definitely looks cleaner and shorter :)